i18n/qgis_fr.ts

```diff
@@ -13360,12 +13360,12 @@ Veuillez le configurer avant d&apos;exécuter les algorithmes LAStools.</transla
     <message>
         <location filename="../src/ui/qgisapp.ui" line="2440"/>
         <source>Add Circular String</source>
-        <translation type="unfinished"></translation>
+        <translation>Ajouter une courbe</translation>
```

Comment:
@DelazJ , should the french translation be capitalized the same way the original english string is?

Reply:
Capitalization works differently in French where only the first word of a sentence or a title should be capitalized. That being said, these days all capitalized titles are more and more seen due to the influence of English.

Reply:
QGIS guidelines requires indeed book capitalization for dialog title. I don't know what this style represents/means in English and don't know whether it has an equivalent in French nor how it applies (i must confess that it looks ugly to me in some cases - here, it would be ok). 
@jfbourdon any reference to French rules for the dialog title (not following book style), please?

Reply:
Feel free to fix things up with a new PR. Better mis-capitalized translations than English ;)

Reply:
@DelazJ +1 with @jfbourdon In French we don't capitalize the title, it's an anglicism.
You can read this [pdf](https://jacques-andre.fr/faqtypo/lessons.pdf) (maybe the best reference) for know your orthotyping. See part 3 for capital letters.

Reply:
Sorry for the delay. Capitalization in French is not always a hard rule, especially for titles. However, there is guidelines. As I am from Quebec, here is a reference from the [Office québécois de la langue française](http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?t1=1&id=1280).

Reply:
Merci à vous deux pour ces références; toujours bien de lire des bases conceptuelles sur ce qu'on fait :+1: